extensions/2.0/Vendor/EXT_mesh_features/README.md

```diff
@@ -441,3 +441,8 @@ Composite|A glTF containing a 3D mesh (house), a point cloud (tree), and instanc
   * Renamed `featureTable` → `propertyTable` and `featureTexture` → `propertyTexture`
   * Removed `featureIdAttributes` and `featureIdTextures`, replaced with `featureIds`
   * Removed string ID references to property tables and textures, replaced with integer IDs
+  * Removed `optional` and `default` and rephrased with properties `noData` and `required`:
+    * Properties are now assumed to be optional unless `required` is true
+    * `noData` is used to specify a sentinel value that indicates missing data
+  * Feature ID values `>= count` now indicate "no associated feature"
+  * `NaN` and `Infinity` are now explicitly disallowed as property values.
```

Comment:
```suggestion
  * `NaN` and `Infinity` are now explicitly disallowed as property values.
  * Byte offsets for buffer views in a GLB-stored BIN chunk are no longer different from the core glTF specification.
```

We removed this line earlier:

> If the buffer view's buffer is the GLB-stored BIN chunk the byte offset is measured relative to the beginning of the GLB. Otherwise it is measured relative to the beginning of the buffer.